suite-native/biometrics/src/components/BiometricsModalRenderer.tsx

```diff
@@ -1,12 +1,15 @@
-import { useBiometrics } from '../useBiometrics';
 import { useIsBiometricsOverlayVisible } from '../biometricsAtoms';
+import { useBiometrics } from '../useBiometrics';
 import { BiometricOverlay } from './BiometricOverlay';
 
 export const BiometricsModalRenderer = () => {
-    useBiometrics();
     const { isBiometricsOverlayVisible } = useIsBiometricsOverlayVisible();
+    const { isBiometricsAuthenticationAllowed, requestAuthenticationCheck } = useBiometrics();
 
-    if (!isBiometricsOverlayVisible) return null;
-
-    return <BiometricOverlay />;
+    return isBiometricsOverlayVisible ? (
+        <BiometricOverlay
+            isBiometricsAuthButtonVisible={!isBiometricsAuthenticationAllowed}
+            onBiometricAuthPress={requestAuthenticationCheck}
+        />
+    ) : null;
```

Comment:
nitpick: I know it's a matter of taste and I used to prefer this syntax too but now I see the previous version more readable than ternary. Feel free to ignore my comment.

Reply:
I prefer the ternary in case there are two possible returns. This way I can see both on one place. 

If the return is somewhere in the code before, it takes me more time to find this.